### PR TITLE
[bitnami/jaeger] Chart missing imagePullSecrets and otlp ports

### LINKED ISSUE #20318: [bitnami/jaeger] deployment missing imagePullSecrets
### Name and Version

bitnami/jaeger 1.3.5

### What architecture are you using?

None

### What steps will reproduce the bug?

Using Jaeger chart with values that contain either `global.imagePullSecrets` or `image.pullSecrets`:

```
global:
  imageRegistry: "my.registry.com"
  imagePullSecrets:
    - my-docker-secret

image:
  pullSecrets:
    - my-docker-secret
```

and deploying it or simply using `helm template` will show that
there are no rendered imagePullSecrets for Jaeger agent, collector or query. Only place with imagePullSecrets is the cassandra subchart.

In the deployment.yaml's of agent, collector and query there is no property imagePullSecrets defined.

### Are you using any custom parameters or values?

_No response_

### What is the expected behavior?

It is expected that Jaeger agent, collector and query deployments have imagePullSecrets if specified in values

### What do you see instead?

imagePullSecrets is not rendered even though it is present in values

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by avl-team-pandas**
### Description of the change

- fix missing imagePullSecrets in templates
- expose missing otlp receiver ports in jaeger collector
  -  https://www.jaegertracing.io/docs/1.50/deployment/#collector ports 4317 and 4318 are now exposed 

### Benefits

- images can be pulled from private registry
- jaeger can receive traces using otlp protocol

### Additional Information

- this fixes #20318 

### Checklist

- [X ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you fix the DCO issue?

**Comment by avl-team-pandas** [comment]
Hi @javsalgar ,

DCO is now fixed, I guess it did not work by including it manually in commit message without using the "-s" flag.

**Review by migruiz4 — APPROVED** [review]
Thank you very much for your contribution @avl-team-pandas, changes look good to me!
I have bumped a minor version since this PR adds new features.

**Comment by avl-team-pandas** [comment]
Hi @migruiz4,
thank you for the review. You will also have to merge the PR since you have write access.

**Comment by migruiz4** [comment]
Don't worry, I didn't forget to merge about this. We have to wait for the CI/CD pipelines to succeed after a commit before we can merge 😄 
It is done now, thanks again!

**Pull request merged**
